sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/KalixRunner.scala

```diff
@@ -81,17 +83,40 @@ object KalixRunner {
   def prepareConfig(config: Config): Config = {
 
     val mainConfig = config.getConfig("kalix.system").withFallback(config)
-    val devConfigFile = new File(System.getProperty("user.dir"), "dev-mode.conf")
 
-    Option
-      .when(devConfigFile.exists()) {
-        ConfigFactory.parseFile(devConfigFile)
+    // enrich config with extra dev-mode service port mappings
+    if (mainConfig.hasPath("kalix.dev-mode.service-port-mappings")) {
+
+      val mappings = mainConfig.getConfig("kalix.dev-mode.service-port-mappings")
+      // each kalix.dev-mode.service-port-mappings becomes a new akka.grpc.client service-discovery mapping
+      // that is then appended to the main configuration
+      mappings.entrySet().asScala.foldLeft(mainConfig) { (main, entry) =>
+        val serviceName = entry.getKey
+        val port = entry.getValue.render()
+
+        logger.info(s"Adding dev-mode service port mapping for $serviceName:$port")
```

Comment:
Keep me honest here. Because it's `info` I understand that the message is meant for the user but on the other hand talking about `mapping`, it seems to me is meant for a developer. From the comment above I think it's creating the configuration to create a gRPC client to consume some service. 
If the message it's for the user do you think is worth giving some other info?

Reply:
Correct. It might be better to use debug. 

The goal here is indeed to print it in during developer. 

True that some one can still build a docker image that by default such a mapping and then deploy it to prod. That won't work and the user/developer will be explicitly taking a path that will cause malfunction.

Anyway, the plugin is already printing the dev-mode port mappings, so we can probably remove this logging all together instead of using `debug`.

Reply:
I see. I understand it better now 👍 
One less line of code sounds like a plan to me.